tests/python/PythonPackageInitBearTest.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import os
+import shutil
+
+from queue import Queue
+
+from coalib.settings.Section import Section
+
+from bears.python.PythonPackageInitBear import PythonPackageInitBear
+
+
+class PythonPackageInitBearTest(unittest.TestCase):
+
+    def setUp(self):
+
+        if not os.path.exists('with-init'):
+            os.mkdir('with-init')
+        open('with-init/__init__.py', 'a').close()
+
+        if not os.path.exists('not-ignored-no-init'):
+            os.mkdir('not-ignored-no-init')
+        open('not-ignored-no-init/test-file.py', 'a').close()
```

Comment:
there's no need to actually put files in the FS since you use the file_dict - makes everything way simpler to test :)